qiskit_experiments/library/characterization/resonator_spectroscopy.py

```diff
@@ -0,0 +1,218 @@
+# that they have been altered from the originals.
+
+"""Spectroscopy experiment class for resonators."""
+
+from typing import Iterable, Optional, Tuple
+import numpy as np
+
+from qiskit import QuantumCircuit
+from qiskit.circuit import Parameter
+from qiskit.exceptions import QiskitError
+from qiskit.providers import Backend
+import qiskit.pulse as pulse
+
+from qiskit_experiments.framework import Options
+from qiskit_experiments.library.characterization.spectroscopy import Spectroscopy
+from .analysis.resonator_spectroscopy_analysis import ResonatorSpectroscopyAnalysis
+
+
+class ResonatorSpectroscopy(Spectroscopy):
+    """Perform spectroscopy on the readout resonator.
+
+    # section: overview
+        This experiment does spectroscopy on the readout resonator. It applies the following
+        circuit
+
+        .. parsed-literal::
+
+                 ┌─┐
+              q: ┤M├
+                 └╥┘
+            c: 1/═╩═
+                  0
+
+        where a spectroscopy pulse is attached to the measurement instruction.
+
+        Side note: when doing readout resonator spectroscopy, each measured IQ point has a
+        frequency dependent phase. Close to the resonance, the IQ points start rotating around
+        in the IQ plan. This effect must be accounted for in the data processing to produce a
+        meaningful signal. The default data processing workflow will therefore reduce the two-
+        dimensional IQ data to one-dimensional data using the magnitude of each IQ point.
+
+    # section: analysis_ref
+        :py:class:`ResonatorSpectroscopyAnalysis`
+
+    # section: see_also
+        qiskit_experiments.library.characterization.qubit_spectroscopy.QubitSpectroscopy
+    """
+
+    @classmethod
+    def _default_experiment_options(cls) -> Options:
+        """Default option values used for the spectroscopy pulse.
+
+        All units of the resonator spectroscopy experiment are given in seconds.
+
+        Experiment Options:
+            amp (float): The amplitude of the spectroscopy pulse. Defaults to 1 and must
+                be between 0 and 1.
+            acquire_duration (float): The duration of the acquisition instruction. By
+                default is lasts 240 ns, i.e. the same duration as the measurement pulse.
+            acquire_delay (float): The duration by which to delay the acquire instruction
+                with respect to the measurement pulse.
+            duration (float): The duration in seconds of the spectroscopy pulse.
+            sigma (float): The standard deviation of the spectroscopy pulse in seconds.
+            width (float): The width of the flat-top part of the GaussianSquare pulse in
+                seconds. Defaults to 0.
+        """
+        options = super()._default_experiment_options()
+
+        options.amp = 1
+        options.acquire_duration = 240e-9
+        options.acquire_delay = 0
+        options.duration = 240e-9
+        options.sigma = 60e-9
```

Comment:
```suggestion
        options.acquire_duration = 240e-9
        options.acquire_delay = 0
        options.duration = 240e-9
        options.sigma = 60e-9
```
I feel like this is bit weird since qubit spectroscopy takes those values in sec.

Reply:
Good point. Fixed here: 8eb9d20